<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Config</class>
 <widget class="QWidget" name="Config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Type of the Similarity Measure</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="MeasureType">
          <item>
           <property name="text">
            <string>Complete</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nearest Neighbors</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>File for the results</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KUrlRequester" name="Results">
          <property name="toolTip">
           <string>Name of the file where the results should be saved [Results].</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Export Matrixes</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="ExportDocsSims">
          <property name="text">
           <string>Documents Similarities</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KUrlRequester" name="DocsSims"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="ExportDocsIncs">
          <property name="text">
           <string>Documents Inclusions</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KUrlRequester" name="DocsIncs"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Documents</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="Docs">
        <property name="text">
         <string>Statistics on documents [Docs]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ProfDoc">
        <property name="text">
         <string>Statistics on profiles and documents [ProfDoc]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="GroupDoc">
        <property name="text">
         <string>Statistics on groups and documents [GroupDoc]</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Profiles</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="Profiles">
        <property name="text">
         <string>Statistics on profiles [Profiles]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="SameDocProf">
        <property name="text">
         <string>Statistics on documents assessed by several profiles [SameDocProf]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="GroupProf">
        <property name="text">
         <string>Statistics on groups and profiles [GroupProf]</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>ExportDocsSims</sender>
   <signal>toggled(bool)</signal>
   <receiver>DocsSims</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ExportDocsIncs</sender>
   <signal>toggled(bool)</signal>
   <receiver>DocsIncs</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
